docs/maintainers/faqs.md

```diff
@@ -131,6 +131,16 @@ Short answer: no. For the moment, we have all the v3 tracks in this single monor
 
 The basic communication channel for maintainers is Slack, and in particular the maintainer-only `#maintaining-<track>` channel. For communication with everyone else, GitHub issues should be used, for obvious reasons (discoverability, familiarity, etc.). We've also enabled the GitHub Projects feature on the v3 repo, so you can organize your track's v3 transition there too.
 
+## Should I use citations
+
+Yes! Please use citations in any reference documentation (files under `**/reference/**`). We are using the [Science of Computer Programming citation style][socp-citation-style]. This means that when citing website, please use this format:
+
+> $TITLE, $SOURCE. ($PUBLISHED_DATE). <$URL> (accessed $ACCESSED_DATE).
+
+For example:
+> Actor model, Wikipedia. (2020). <https://en.wikipedia.org/w/index.php?title=Actor_model&oldid=939106706> (accessed February 29, 2020).
```

Comment:
```suggestion
> $TITLE, $SOURCE. ($PUBLISHED_DATE). $URL (accessed $ACCESSED_DATE).

For example:
> Actor model, Wikipedia. (2020). https://en.wikipedia.org/w/index.php?title=Actor_model&oldid=939106706 (accessed February 29, 2020).
```

The `<>` are not part of it, according to your source and Zotero's citation exporter.

Reply:
Markdown specification states: "The link URL may, optionally, be surrounded by angle brackets"

In doing so, it should link and show itself as the link.  Perhaps this is why it is there.  And easier to put into place from the tooling, when dropping it into markdown, than doing the \[url](url) dance.

Reply:
It shows as link without them too (at least on GitHub), and manually adding `<>` whenever citing something is rather tedious.

Reply:
> It shows as link without them too (at least on GitHub), and manually adding `<>` whenever citing something is rather tedious.

Be a wee bit careful relying on Github's behavior there, I've seen quite a few situations in the wild where other markdown renderers (notably CommonMark-compliant ones) have left "naked" links entirely un-clickable.

Reply:
Ok, I found a way to adapt the style to auto-generate the citation with `<>`, so I'm fine with this.

Reply:
Yeah, I think that this totally depends on what tooling Exercism is using...

Reply:
I was also confused about the `<>` bit. I'd much prefer to use the exact format used on the website we link to.

@kotp The tooling we're using for the website is [docsify](https://github.com/exercism/v3/blob/master/docs/maintainers/website.md). I've just checked if the variant that does not use `<>` characters works there, and it does! So my suggestion would be to apply @SaschaMann's suggestion and remove these `<>` characters.